Content.Client/Stylesheets/StyleNano.cs

```diff
@@ -1341,6 +1345,11 @@ public StyleNano(IResourceCache resCache) : base(resCache)
                     new StyleProperty(Label.StylePropertyAlignMode, Label.AlignMode.Center),
                 }),
 
+                new StyleRule(new SelectorElement(typeof(PanelContainer), new[] { OptionButton.StyleClassBackPanel }, null, null), new[]
+                {
+                    new StyleProperty(PanelContainer.StylePropertyPanel, new StyleBoxFlat { BorderColor = OptionButtonBorder, BackgroundColor = OptionButtonBackground, BorderThickness = new Thickness(2, 2, 2, 2) }),
+                }),
```

Comment:
Awful copypaste. Use the helpers, do not construct StyleRule manually.

Reply:
Done.
But now I really want to rewrite the code above that doesn't use helpers 😅

Reply:
I'm [already doing that](https://github.com/space-wizards/space-station-14/pull/28356), don't worry about it.